SCons/Script/SConscript.xml

```diff
@@ -372,7 +400,7 @@ Return('val1 val2')
 
 <scons_function name="SConscript">
 <arguments>
-(scripts, [exports, variant_dir, duplicate, must_exist])
+(scriptname, [exports, variant_dir, duplicate, must_exist])
```

Comment:
scriptnames? It can be plural right?

Reply:
the actual arg is `*ls` ?

Reply:
was trying to match up with wording. as noted elsewhere, unnamed args aren't really helpful in doc generation etc. but we have a good number spread around. SConscript is a weird function, can take 0, 1 or 2 positional args. the 0 part being a bit odd...

Reply:
and yes, pluralizing it is ok, if you prefer that.

Reply:
I think pluralizing would be better. though I think rarely used, it is accurate.